packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -512,32 +512,36 @@ void main() {
       ),
     );
 
-    // TODO(goderbauer): traversal order is incorrect, https://github.com/flutter/flutter/issues/14375
```

Comment:
Unfortunately, this test still doesn't verify that the traversal order is defined correctly -- and I believe we currently have no way of asserting traversal order in a widget test like this because at this point the SemanticNodes have not been sorted yet?

Reply:
It does now. After https://github.com/flutter/flutter/pull/16253/commits/fad0555668dd95bcf2177dc5762d72a678c85060 `hasSemantics` tests traversal order by default.

Reply:
Perfect. 👍